.travis.yml

```diff
@@ -13,3 +13,4 @@ env:
   global:
```

Comment:
okay so the `secure` token is obviously secure. and its not idea but okay to have our hostname in there.